prov/psm2/src/psmx2_init.c

```diff
@@ -70,9 +70,39 @@ int	 psmx2_tag_layout_locked = 0;
 
 static void psmx2_init_env(void)
 {
+	char *ompi_job_key;
+	psm2_uuid_t uuid = {};
+	unsigned long long *u = (unsigned long long *)uuid;
+
 	if (getenv("OMPI_COMM_WORLD_RANK") || getenv("PMI_RANK") || getenv("PMIX_RANK"))
 		psmx2_env.name_server = 0;
 
+	/*
+	 * Check for Open MPI job key. If set, convert it to the default uuid
+	 * string. This will be overridden by the FI_PSM2_UUID variable, and
+	 * both will have lower priority than the auth_key passed via ep_attr.
+	 *
+	 * Open MPI 4.x:
+	 * OMPI_MCA_orte_precondition_transports=e7afb9969e40afd7-9183d92671c847de
+	 *
+	 * Open MPI 5.x:
+	 * OMPI_MCA_precondition_transports=e7afb9969e40afd7-9183d92671c847de
+	 */
+
+	ompi_job_key = getenv("OMPI_MCA_precondition_transports");
```

Comment:
Ah crumb - I didn't realize that prior versions of OMPI had the "orte" in the envar name. I'll update PMIx so it matches - no intent to change the name.

Reply:
Fixed - the envar in OMPI v5 now matches all prior versions.

Reply:
Thanks. PR is updated to reflect the change.